lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/service/LessonBuilderEntityProducer.java

```diff
@@ -510,6 +515,28 @@ else if (item.getType() == SimplePageItem.ASSESSMENT)
 		    addAttr(doc, itemElement, "samewindow", item.isSameWindow() ? "true" : "false");
 		
 		String attrString = item.getAttributeString(); //json encoded attributes
+
+		final JSONParser jsonParser = new JSONParser(); // JSON parser used to parse the attribute string into a JSON object
```

Comment:
Why not use j2ee JsonObject here, instead? I know simplepagebean uses Google's lib, but the j2ee version is a very similar api, with perhaps better builder patterns.